test_scripts/Polices/External_UCS_Informing_HMI/018_ATF_N_GetListofPermissions_empty_due_to_missing_status_OnAppPermissionChange.lua

```diff
@@ -0,0 +1,125 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] [External UCS] SDL informs HMI about <externalConsentStatus> via GetListOfPermissions response
+-- [HMI API] GetListOfPermissions request/response
+-- [HMI API] ExternalConsentStatus struct & EntityStatus enum
+--
+-- Description:
+-- For Genivi applicable ONLY for 'EXTERNAL_PROPRIETARY' Polcies
+-- Check that SDL invalidates notification OnAppPermissionConsent due to missing mandatory parameter entityType
+--
+-- 1. Used preconditions
+-- SDL is built with External_Proprietary flag
+-- SDL and HMI are running
+-- Application is registered and activated
+-- PTU file is updated and application is assigned to functional groups: Base-4, user-consent groups: Location-1 and Notifications
+-- PTU has passed successfully
+-- HMI sends <externalConsentStatus> to SDl via OnAppPermissionConsent (mandatory parameter entityType is missed, rest of params present and within bounds, EntityStatus = 'ON')
```

Comment:
@vvvakulenko, according to test set, in this script EntityStatus should be missing, other parameters should be present. Please update description.